i18n/en/docusaurus-plugin-content-docs/version-5.0/13-sdk/01overview.md

```diff
@@ -0,0 +1,94 @@
+
+
+## Remoting SDK
+
+The Remoting protocol SDK, as the initial SDK evolution of Apache RocketMQ, uses the Remoting communication protocol of internal components of RocketMQ. It's used for communication of internal components of services and also supports API communication for client message sending and control operations.
+
+### SDK Info
+
+The currently supported programming languages and code repositories are as follows:
+
+| **Language** |**ReleaseNote** |**SDK Repo** |
+|---------|---------|---------|
+|Java|[ReleaseNote from main repo](https://github.com/apache/rocketmq/releases)|[Main repo](https://github.com/apache/rocketmq)|
+|Go|[ReleaseNote from main repo](https://github.com/apache/rocketmq-client-go/releases)|[Main repo](https://github.com/apache/rocketmq-client-go)|
+
+### Features
+Waiting updates.
+
+
+## gRPC SDK
+The gRPC protocol SDK, introduced in Apache RocketMQ version 5.0, aims to provide a cloud-native, robust client solution for all major programming languages, including Java, C++, C#, Golang, JavaScript, and Rust. The gRPC SDK adheres to the [rocketmq-apis](https://github.com/apache/rocketmq-apis) constraints and uses Protocol Buffers and gRPC to replace the old protocol in version 4.x.
+
+The gRPC SDK has the following advantages over the earlier Remoting protocol SDK:
+- More concise interface design, easier to understand and less prone to error.
+- Better interface design with clear parameters and exception types.
+- Immutable interface design to avoid business exceptions caused by parameter and information leakage.
+- Better support for multiple languages, as the gRPC protocol has the advantage of supporting multiple languages and enables lower cost evolution to achieve consistent behavior in multiple language SDKs.
+
+Want the detailed design thinking and evolutionary direction, refer to [RIP-37: New and Unified APIs](https://shimo.im/docs/m5kv92OeRRU8olqX) and [RIP-39: Support gRPC protocol](https://shimo.im/docs/gXqmeEPYgdUw5bqo).
+
+### SDK Info
+
+The currently supported programming languages and code repositories are as follows:
+
+| **Language** |**ReleaseNote** |**SDK Repo** |
+|---------|---------|---------|
+|Java|[ReleaseNote from main repo](https://github.com/apache/rocketmq-clients/releases)|[Main repo](https://github.com/apache/rocketmq-clients)|
+|C++|[ReleaseNote from main repo](https://github.com/apache/rocketmq-clients/releases)|[Main repo](https://github.com/apache/rocketmq-clients)|
+|.NET|[ReleaseNote from main repo](https://github.com/apache/rocketmq-clients/releases)|[Main repo](https://github.com/apache/rocketmq-clients)|
+|Go|[ReleaseNote from main repo](https://github.com/apache/rocketmq-clients/releases)|[Main repo](https://github.com/apache/rocketmq-clients)|
+|Rust|[ReleaseNote from main repo](https://github.com/apache/rocketmq-clients/releases)|[Main repo](https://github.com/apache/rocketmq-clients)|
```

Comment:
Rust SDK has not been released yet, but the release process will be handled recently.